### ISSUE TITLE
Need a function to compute the translational velocity of a point attached to a body

### ISSUE THREAD

**Issue by hongkai-dai** [RobotLocomotion/drake#11725]
Currently we have a function `Frame::CalcSpatialVelocity` to compute the spatial velocity of a frame, but we don't have the function to compute the translational velocity of a point attached to a frame (I can compute it as v + p.cross(omega), but an API would be nicer).

**Comment by amcastro-tri** [comment]
@hongkai-dai, you can use `SpatialVelocity::Shift()` which accomplishes exactly what you want. Please take a look at its docs and let me know if it solves your problem

**Comment by hongkai-dai** [comment]
@amcastro-tri, that's what I did in my dev branch. But I would say it is not very convenient, especially if the  expressed frame of the shift vector `p_BpBq_E` is different from `B`. When that happens, I need to first compute the pose of `B` relative to `E`, then transform `p_BpBq_B` to `p_BpBq_E`, and then do the shift. It would be more convenient for the user, if there is a function `CalcPointsTranslationalVelocities()`, as a counter part of the existing function `CalcPointsPositions()`.

Anyway, this is not urgent. I called Shift function, just as you suggested. But it would be nicer to provide the API.
